scss/application.scss

```diff
@@ -1,3 +1,4 @@
 @import 'equalizer';
-@import 'styles';
 @import 'titles';
+@import 'styles';
```

Comment:
I'd place titles after this, since it's more specific it makes more sense to have it last.